stl/inc/format

```diff
@@ -2493,15 +2493,15 @@ _NODISCARD _OutputIt _Fmt_write(
         _THROW(format_error("const void* cannot be localized"));
     }
 
-    // Compute the bit width of the pointer (i.e. how many bits it takes to be represented).
-    // Add 3 to the bit width so we always round up on the division.
-    // Divide that by the amount of bits a hexit represents (log2(16) = log2(2^4) = 4).
-    // Add 2 for the 0x prefix.
-    auto _Width = 2 + static_cast<int>(_STD bit_width(reinterpret_cast<uintptr_t>(_Value)) + 3) / 4;
-
     // Since the bit width of 0 is 0, special case it instead of complicating the math even more.
-    if (_Value == nullptr) {
-        _Width = 3;
+    int _Width = 3;
+

```

Comment:
Nitpick: I would purge that newline. It is essentially a single assignment

Reply:
Done!